### ISSUE TITLE
power-distribution-card not accessible on Android Home Assistant Companion App

### ISSUE THREAD

**Issue by T1ppes** [JonahKr/power-distribution-card#59]
I have just updated from version 1.7 to 2.2. Following I removed the power distribution card in my lovelace setup and created a new one using the UI on the webbrowser of my computer.
Using the Android Home Assistant Companion App however I have found out, that this card is not displayed. It seems that also the option to create the power distribution card is not available on the Android App. Can you imagine a reason for this?
In case you require any further information to analyse this problem, let me know and I am glad to share.

**Comment by JonahKr** [comment]
Hey there 👋 @T1ppes 

Since you cannot choose to add it from the Companion app, i would guess it has something to do with the homeassistant loading of the rescources. Uninstall the card and reinstall it and that should do it. If its still not avaiable, try Rebooting Homeassistant. On a Desktop you can try to press "shift + F5" to Hard Reload the Site. 

When you Inspect (Right-Click the site and choose *Inspect* and a tab should show up) the Site and switch to the `console`, you can check if the right version is loaded. It should look like this:
![image](https://user-images.githubusercontent.com/38377070/110333033-66afb780-8021-11eb-9ab8-b2c09dd7b2be.png)

If none of this works, please come back to me with your configuration and i will try to figure it out.

Take Care and stay safe 👍 

Jonah

**Comment by T1ppes** [comment]
Thanks for the quick feedback. So I assume the app has cached some old information. Unfortunatley I cannot inspect anything within the app. Accessing lovelace from my browser on a PC everything is working just fine.
1. I deleted the power-distribution card, rebooted Homeassistant, rebooted my phone and reinstalled the add-on.
2. I cleared the cache on the app.
3. I installed the add-on using the app, following a reset is performed.
All steps did not bring the card back to life on the companion app.
